modules/oc-mirror-migration-differences.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * oc-mirror-migration-v1-to-v2.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-migration-differences_{context}"]
+= Differences between oc-mirror plugin v1 and v2
+
+Before migrating from oc-mirror plugin v1 to v2, you can check differences between oc-mirror plugin v1 to v2. Review the following:
```

Comment:
I like the highlights you used in this section. It makes it easy to spot what needs to change.
I would even use the bold text to highlight the change to make. But I don't know how that would look, or if it would help make it more readable..
Instead of `differences` , maybe we need to talk about changes from v1 to v2?